### PR TITLE
[VLAN] M3-4542: VLAN Detail Skeleton

### PR DESCRIPTION
**Pull request by tiffwong**
Basic VLAN detail skeleton with linode-related parts for the VlanDetailEntity component commented out

### CODE REVIEW

**Review by Jskobos — APPROVED** [review]
It's a little hard to review this, "skeleton" can mean a few things 😆 . The routing needs a small fix, it couldn't hurt to clean up some of the commented out code that we're sure we don't need, but otherwise looks very skeleton-y 👍

File: packages/manager/src/features/Vlans/VlanDetail/VlanDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    loading: false,
+    lastUpdated: 1234,
+    error: undefined
+  };
+
+  return (
+    <React.Fragment>
+      <VlanEntityDetail />
+      <div className={classes.table}>
+        <LandingHeader
```
Comment (WilkinsKa1) [inline packages/manager/src/features/Vlans/VlanDetail/VlanDetail.tsx:69]: We should use the Entity header component (and pass it the `isSecondary` prop- can see a similar setup under LinodeNetworking_CMR)

**Review by WilkinsKa1 — APPROVED** [review]
Non-blocking comment, looks good!

File: packages/manager/src/features/Vlans/VlanDetail/VlanDetailRow.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <Grid container wrap="nowrap" alignItems="center">
+          <Grid item className="py0">
+            <EntityIcon variant="linode" />
+          </Grid>
+          <Grid item>
+            <Typography variant="h3">Label</Typography>
+          </Grid>
+        </Grid>
+      </TableCell>
+      <TableCell>Region</TableCell>
```
Comment (DevDW) [inline packages/manager/src/features/Vlans/VlanDetail/VlanDetailRow.tsx:27]: Status

File: packages/manager/src/features/Vlans/VlanDetail/VlanDetailRow.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      <TableCell>Linodes</TableCell>
```
Comment (DevDW) [inline packages/manager/src/features/Vlans/VlanDetail/VlanDetailRow.tsx:28]: VLAN IP

**Review by DevDW — APPROVED** [review]

**Pull request merged**